chainerx_cc/chainerx/native/native_float16_test.cc

```diff
@@ -0,0 +1,168 @@
+#include "chainerx/native/native_float16.h"
+
+#include <cmath>
+#include <limits>
+#include <vector>
+
+#include <gtest/gtest.h>
+
+namespace chainerx {
+namespace native {
+namespace {
+
+void CheckNear(double expected, double actual, double tol) {
```

Comment:
* Why the tolerance is not taken into account in `== 0` case?
* What's the difference between within (-1, 1) and the other case?

Please write comments.